src/parser/ts/parser.unit.test.ts

```diff
@@ -449,3 +450,54 @@ describe('check if expression has aggregates', () => {
     expect(hasAggregates('ts{granularity="1s"}')).toBeTruthy();
   });
 });
+
+describe('check if default aggregation and granularity will be added', () => {
+  const defaults = { aggregate: 'average', granularity: '1h' };
+  it('should add default aggregate and granularity', () => {
+    const stsWithoutAggregation = STS([Filter('name', 'name')]);
+    const withoutAggregates = injectAggregatesToParsed(stsWithoutAggregation, defaults);
+
+    expect(withoutAggregates).toEqual(
+      STS([Filter('name', 'name'), Filter('aggregate', 'average'), Filter('granularity', '1h')])
+    );
+  });
+  it('should add default granularity only', () => {
+    const stsWithAggregate = STS([Filter('name', 'name'), Filter('aggregate', 'interpolation')]);
+    const withAggregates = injectAggregatesToParsed(stsWithAggregate, defaults);
+
+    expect(withAggregates).toEqual(
+      STS([
+        Filter('name', 'name'),
+        Filter('aggregate', 'interpolation'),
+        Filter('granularity', '1h'),
+      ])
+    );
+  });
+  it('should add default aggregation only', () => {
+    const stsWithGranularity = STS([Filter('name', 'name'), Filter('granularity', '1m')]);
+    const withGranularity = injectAggregatesToParsed(stsWithGranularity, defaults);
+
+    expect(withGranularity).toEqual(
+      STS([Filter('name', 'name'), Filter('granularity', '1m'), Filter('aggregate', 'average')])
+    );
+  });
+  it('should ignore default values', () => {
+    const stsWithAggregateAndGranularity = STS([
+      Filter('name', 'name'),
+      Filter('granularity', '1m'),
+      Filter('aggregate', 'interpolation'),
+    ]);
+    const withAggregatesAndGranularity = injectAggregatesToParsed(
+      stsWithAggregateAndGranularity,
+      defaults
+    );
+
+    expect(withAggregatesAndGranularity).toEqual(stsWithAggregateAndGranularity);
+  });
+  it('should not add default values in case of default aggregate is none', () => {
+    const sts = STS([Filter('name', 'name')]);
+    const stsWithInjected = injectAggregatesToParsed(sts, { aggregate: 'none', granularity: '2h' });
```

Comment:
this should be removed now

Reply:
why? to set aggregation to `none` is the only way to fetch real data instead of aggregated

Reply:
I though we could operate with undefined/null value instead.
but ok, this approach works as well

Reply:
I did this in this way cause we have `none` value in 1st tab, just to keep consistency when we're switching tabs